server/src/main/java/com/cloud/user/AccountManagerImpl.java

```diff
@@ -1633,16 +1720,16 @@ public AccountVO updateAccount(UpdateAccountCmd cmd) {
         }
 
         // Check if user performing the action is allowed to modify this account
-        checkAccess(CallContext.current().getCallingAccount(), _domainMgr.getDomain(account.getDomainId()));
+        checkAccess(getCurrentCallingAccount(), _domainMgr.getDomain(account.getDomainId()));
 
         // check if the given account name is unique in this domain for updating
         Account duplicateAcccount = _accountDao.findActiveAccount(newAccountName, domainId);
         if (duplicateAcccount != null && duplicateAcccount.getId() != account.getId()) {// allow
-                                                                                        // same
-                                                                                        // account
-                                                                                        // to
-                                                                                        // update
-                                                                                        // itself
+            // same
```

Comment:
Would be better if we combine all the words in a single line.

Reply:
Well, these were already there. Probably the lines shifted because of my changes, but I can fix it without problems.

Side note: I do not appreciate code comments. They are a sign that the code should be extracted to smaller methods and then documented.